finetrainers/trainer.py

```diff
@@ -547,10 +556,18 @@ def train(self) -> None:
         }
         logger.info(f"Training configuration: {json.dumps(info, indent=4)}")
 
-        # TODO(aryan): handle resume from checkpoint
         global_step = 0
         first_epoch = 0
         initial_global_step = 0
+        
+        # Potentially load in the weights and states from a previous save
+        initial_global_step, global_step, first_epoch = sort_out_and_load_latest_ckpt_states(
```

Comment:
`sort_out_and_load_latest_ckpt_states()` and `save_intermediate_ckpt_states()` are unlikely to change for models and training algos.

Reply:
Yes, lgtm